views/movies/new-movie.hbs

```diff
@@ -0,0 +1,28 @@
+<h2>Add Movie</h2>
+
+<form action="/movies/create" method="post">
+
+<label>Title:
+    <input type="text" name="title">
+</label>
+
+<label>Genre:
+    <input type="text" name="genre">
+</label>
+
+<label>Plot:
+    <input type="text" name="plot">
+</label>
+
+<label>
+    <select name="cast" id="cast">
+        <option value="Cameron Diaz">Cameron Diaz</option>
+        <option value="Brad Pitt">Brad Pitt</option>
+        <option value="henry cavill">henry cavill</option>
+        <option value="lebron james">lebron james</option>
+        <option value="Ana de armas">Ana de armas</option>
+    </select>
+</label>
```

Comment:
You should nto be hardcoding the list. Because if I create new celebrity-> it won't be in my dropdown. Instead you want something like this:

```
<select name="cast">
    {{#each celebrities}}
        <option value="{{_id}}">{{name}}</option>
     {{/each}}
</select>
```